airflow-core/src/airflow/ui/src/pages/Run/Header.tsx

```diff
@@ -119,7 +120,15 @@ export const Header = ({
             : [
                 {
                   label: translate("dagRun.triggeringUser"),
-                  value: <Text>{dagRun.triggering_user_name}</Text>,
+                  value: (
+                    <Link asChild color="fg.info">
+                      <RouterLink
+                        to={`/dag_runs?triggering_user_name_pattern=${encodeURIComponent(dagRun.triggering_user_name)}`}
```

Comment:
We should probably use the constant
```suggestion
                        to={`/dag_runs?${TRIGGERING_USER_NAME_PATTERN}=${encodeURIComponent(dagRun.triggering_user_name)}`}
```

I see that we also don't do it in other redirect link in the application. That could be a nice follow up @dheerajturaga . (go over all `to=\{...` link and replace query params hard coded string by their CONST value from `SearchParamsKeys`)

Reply:
Marking as resolved so we can merge this one.

Reply:
I addressed this in #55218